Userland/Services/WindowServer/Compositor.cpp

```diff
@@ -913,6 +929,48 @@ void Compositor::change_cursor(const Cursor* cursor)
     }
 }
 
+void Compositor::render_overlays()
+{
+    // NOTE: overlays should always be rendered to the temporary buffer!
+    for (auto it = m_overlay_list.begin(); it != m_overlay_list.end(); ++it) {
+        auto& overlay = *it;
```

Comment:
Can't this use a range-based loop?
```suggestion
    for (auto& overlay : m_overlay_list) {
```